Open TWIR for 31. Jul 2020 (#380)
giantswarm/releases: announcements/2020-Week-31/azure.md

# :zap: This Week in Azure Releases (July 31, 2020):zap:

> [Azure v12.0.0](https://github.com/giantswarm/releases/blob/master/azure/v12.0.0) contains Kubernetes v1.17, makes NGINX Ingress Controller optional not preinstalled and removes Quay from being a single point of failure by using Docker mirroring feature.

Author: ```suggestion
[Azure v12.0.0](https://github.com/giantswarm/releases/blob/master/azure/v12.0.0) contains Kubernetes v1.17, makes NGINX Ingress Controller optional (not pre-installed), and removes Quay from being a single point of failure by using Docker mirroring feature.
```

**Optional Apps Releases**  
(Can be installed from App Catalogs)

> 1. 

Author: ```suggestion
1. [Prometheus Operator v0.3.4](https://github.com/giantswarm/prometheus-operator-app/blob/master/CHANGELOG.md#034---2020-07-22) is upgraded to the latest verision with minor bug fixes.
```

> 1. [Prometheus Operator v0.3.4](https://github.com/giantswarm/prometheus-operator-app/blob/master/CHANGELOG.md#034---2020-07-22) is upgraded to the latest verision with minor bug fixes.

Author: ```suggestion
1. [NGINX IC v1.8.1](https://github.com/giantswarm/nginx-ingress-controller-app/blob/master/CHANGELOG.md#181---2020-07-28) made node ports configurable for NodePort Service type. (Note: NGINX is Optional and not Pre-Installed in AWS 10+ clusters)
2. [NGINX IC v1.8.0](https://github.com/giantswarm/nginx-ingress-controller-app/blob/master/CHANGELOG.md#180---2020-07-24) supported multiple NGINX IC App installations per tenant cluster and dropped deprecated configuration properties. (Note: NGINX is Optional and not Pre-Installed in AWS 10+ clusters)
3. [Prometheus Operator v0.3.4](https://github.com/giantswarm/prometheus-operator-app/blob/master/CHANGELOG.md#034---2020-07-22) is upgraded to the latest verision with minor bug fixes.
```

Author: @sslavic since NGINX is now optional in Azure, we should include optional NGINX releases in Azure announcement too, right? 🤔

# :zap: This Week in Azure Releases (July 31, 2020):zap:

[Azure v12.0.0](https://github.com/giantswarm/releases/blob/master/azure/v12.0.0) contains Kubernetes v1.17, makes NGINX Ingress Controller optional (not pre-installed), and removes Quay from being a single point of failure by using Docker mirroring feature.

**Optional Apps Releases**  
(Can be installed from App Catalogs)

> 1. [NGINX IC v1.8.1](https://github.com/giantswarm/nginx-ingress-controller-app/blob/master/CHANGELOG.md#181---2020-07-28) made node ports configurable for NodePort Service type. (Note: NGINX is Optional and not Pre-Installed in Azure 12+ clusters)
> 2. [NGINX IC v1.8.0](https://github.com/giantswarm/nginx-ingress-controller-app/blob/master/CHANGELOG.md#180---2020-07-24) supported multiple NGINX IC App installations per tenant cluster and dropped deprecated configuration properties. (Note: NGINX is Optional and not Pre-Installed in Azure 12+ clusters)
> 3. [Prometheus Operator v0.3.4](https://github.com/giantswarm/prometheus-operator-app/blob/master/CHANGELOG.md#034---2020-07-22) is upgraded to the latest verision with minor bug fixes.

Reviewer: ```suggestion
1. [NGINX IC v1.8.3](https://github.com/giantswarm/nginx-ingress-controller-app/blob/master/CHANGELOG.md#183---2020-07-31) fixes controller RBAC permissions, granting "get" and "update" of leader election ConfigMap lock.. (Note: NGINX is Optional and not Pre-Installed in AWS 10+ clusters)
2. [NGINX IC v1.8.1](https://github.com/giantswarm/nginx-ingress-controller-app/blob/master/CHANGELOG.md#181---2020-07-28) made node ports configurable for NodePort Service type. (Note: NGINX is Optional and not Pre-Installed in Azure 12+ clusters)
3. [NGINX IC v1.8.0](https://github.com/giantswarm/nginx-ingress-controller-app/blob/master/CHANGELOG.md#180---2020-07-24) supported multiple NGINX IC App installations per tenant cluster and dropped deprecated configuration properties. (Note: NGINX is Optional and not Pre-Installed in Azure 12+ clusters)
4. [Prometheus Operator v0.3.4](https://github.com/giantswarm/prometheus-operator-app/blob/master/CHANGELOG.md#034---2020-07-22) is upgraded to the latest verision with minor bug fixes.
```

# :zap: This Week in Azure Releases (July 31, 2020):zap:

[Azure v12.0.0](https://github.com/giantswarm/releases/blob/master/azure/v12.0.0) contains Kubernetes v1.17, makes NGINX Ingress Controller optional (not pre-installed), and removes Quay from being a single point of failure by using Docker mirroring feature.

**Optional Apps Releases**  
(Can be installed from App Catalogs)

> 1. [NGINX IC v1.8.3](https://github.com/giantswarm/nginx-ingress-controller-app/blob/master/CHANGELOG.md#183---2020-07-31) fixes controller RBAC permissions, granting "get" and "update" of leader election ConfigMap lock.. (Note: NGINX is Optional and not Pre-Installed in Azure 12+ clusters)

Author: ```suggestion
1. [NGINX IC v1.8.3](https://github.com/giantswarm/nginx-ingress-controller-app/blob/master/CHANGELOG.md#183---2020-07-31) fixes controller RBAC permissions, granting "get" and "update" of leader election ConfigMap lock. (Note: NGINX is Optional and not Pre-Installed in Azure 12+ clusters)
```
